### ISSUE TITLE
CI: remove Buildah?

### ISSUE THREAD

**Issue by shanegoff** [hpc/charliecloud#990]
Buildah causes frequent issue for CI. We can keep support in our wrappers without failing all CI due to buildah errors though.

**Comment by rstyd** [comment]
I've been working on fixing buildah errors for the vagrantfile, and it seems like whenever I fix one another pops up like whack-a-mole. I'd support at least not running automated tests on buildah.

**Comment by shanegoff** [comment]
Recently buildah has created a problem with CI:

```
not ok 41 Dockerfile: ENV parsing
# (in test file build/50_dockerfile.bats, line 370)
#   `[[ $status -eq 0 ]]' failed
# building with: buildah
# STEP 1: FROM centos8
# Resolving "centos8" using unqualified-search registries (/etc/containers/registries.conf)
# error creating build container: Error initializing source docker://centos8:latest: Error reading manifest latest in docker.io/library/centos8: errors:
# denied: requested access to the resource is denied
# unauthorized: authentication required
# 
```

**Comment by reidpr** [comment]
Closed by PR #991; forgot to link it correctly.
